### DISCUSSION TITLE
node.js module subscription error 'observableish.subscribe' is not a function

### DISCUSSION THREAD

**Discussion by capymachi** [urql-graphql/urql#2896] [Q&A] [answered]
### Describe the bug

honestly I'm not sure how this is supposed to work at all. Just trying to follow the documentation, and as far as I can tell this is a bug and not user error? Please correct me if I'm wrong, I just want this to work...

```
/home/machi/projects/gate-controller-test/node_modules/@urql/core/dist/urql-core.js:435
        sub = observableish.subscribe({
                            ^

TypeError: observableish.subscribe is not a function
    at /home/machi/projects/gate-controller-test/node_modules/@urql/core/dist/urql-core.js:435:29

Node.js v18.12.1
```

### Reproduction

https://github.com/Muchieman7/urql-subscription

### Urql version

urql 3.0.3

### Validations

- [X] I can confirm that this is a bug report, and not a feature request, RFC, question, or discussion, for which [GitHub Discussions](https://github.com/urql-graphql/urql/discussions) should be used
- [X] Read the [docs](https://formidable.com/open-source/urql/docs/).
- [X] Follow our [Code of Conduct](https://github.com/urql-graphql/urql/blob/main/CODE_OF_CONDUCT.md)

**Comment by kitten** [comment, marked as answer]
I think you've got a simple typo here: https://github.com/Muchieman7/urql-subscription/blob/ddc36fb4ce59a16280bd24a1ab0460b634bfff9b/test.js#L17

Just a heads up, if you have a problem that's likely in the realm of a question and not a bug in the library, we'd prefer to use discussions, but no worries :v: I'll convert this over to one and we can chat more there if needed

**Reply by capymachi** [reply]
Wow - I spent a good while trying to make sure that this wasn't my fault lol. I couldn't trace it back to anything in my code 🥲 I ended up hacking my way through making apollo work without react which was more trouble than it should've been... One typo fix and it's working perfectly! Damned dyslexia. Thanks for the blazing fast reply, I literally missed it because of how fast it was!
